packages/usa-file-input/src/styles/_usa-file-input.scss

```diff
@@ -188,14 +188,14 @@
   pointer-events: none;
 
   .usa-file-input__instructions {
-    opacity: 0.5;
+    opacity: 0.7;
```

Comment:
On second inspection it actually looks like `.7` isn't passing for regular-sized text. I'll have to bump this to `.8`

60% opacity works for most elements but because the background on the file input gets changed to a light gray, we need this text to be darker.

Reply:
What do we think of `.8` and removing the link color from the disabled version?

![image](https://user-images.githubusercontent.com/25211650/211404707-f7bd0a7d-2bf3-4c64-a41f-ec62a65615ca.png)